<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QStackedWidget" name="MainStack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="HomePage">
     <widget class="QPushButton" name="PlayButton">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>350</y>
        <width>170</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QLabel" name="GameTitle">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>140</y>
        <width>550</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;
border-radius: 30px;
font: bold normal large &quot;Comic Sans MS&quot;;
font-size: 48px;
color: rgba(0,0,0,255);
background-color:rgba(100,150,255,100);
background-position: bottom center;
qproperty-alignment: AlignCenter;</string>
      </property>
      <property name="text">
       <string>Platformer Game</string>
      </property>
     </widget>
     <widget class="QLabel" name="Background">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>1600</width>
        <height>620</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>Background</zorder>
     <zorder>PlayButton</zorder>
     <zorder>GameTitle</zorder>
    </widget>
    <widget class="QWidget" name="MapSelect"/>
    <widget class="QWidget" name="MapEditor"/>
    <widget class="QWidget" name="Game"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
